pkg/tsdb/prometheus/testdata/range_simple.result.json

```diff
@@ -25,7 +25,6 @@
         },
         "values": [
           [1641889530.123, "54"],
-          [1641889531.123, "65"],
```

Comment:
where did it go?

Reply:
@ryantxu i think this was removed to simulate a gap in one of the series